apps/ey-pdfgen/.nais/dev.yaml

```diff
@@ -7,8 +7,6 @@ metadata:
     team: etterlatte
 spec:
   image: {{ image }}
-  ingresses:
-    - https://etterlatte-pdfgen.intern.dev.nav.no # For testing fra lokal maskin
```

Comment:
Det er ikke noe vi burde ha for å lage søknaden til OMS?

Reply:
Godt poeng – legger den tilbake.